kotest-property/src/commonMain/kotlin/io/kotest/property/arbitrary/merge.kt

```diff
@@ -38,3 +38,23 @@ fun <A, B : A> Arb<A>.merge(other: Gen<B>): Arb<A> = trampoline { sampleA ->
          }
    }
 }
+
+/**
+ * Returns a new [Arb] which merges the provided arbs.
+ *
+ * Each sample is generated by randomly selecting one of the provided arbs and returning a sample from it.
+ *
+ * @param arbs the args to merge
+ * @return the merged arg.
+ */
+fun <A : Any> Arb.Companion.merge(arbs: List<Arb<A>>): Arb<A> {
```

Comment:
@tbvh I'm not used to the type signature of this merge function. Normally merge would take exactly one covariant type of the receiver. For it to take a list and without recriver, this may need to be renamed like mergeAll or combineAll. You can also put that together in the same file as Arb.choose and mention that it is an alias if choose. Instead, I highly encourage checking out `Arb.choice` or `Arb.choose`. that does this.